style.css

```diff
@@ -764,3 +754,302 @@ button:hover {
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 10%;
+  height: 1000px;
+  margin-top: 11px;
+  top: 1px;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+}
+
+.card-title-mobile > h2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: left;
+  justify-content: left;
+  margin-left: -300px;
+}
+
+.card-links-mobile {
+  display: flex;
+  font-size: 12px;
+  font-weight: 600;
+  gap: 15px;
+  letter-spacing: 0.03em;
+  flex-direction: row;
+  align-items: center;
+  padding: 28px;
+  width: 77px;
+  margin-left: 510px;
+  margin-right: 30px;
+  margin-top: -50px;
+  height: 51px;
+  color: #f6f0f0;
+  border-radius: 8px;
+}
+
+.btn-card-details-mobile {
+  display: flex;
+  gap: 20px;
+  flex-direction: row;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  align-items: center;
+  padding: 18px;
+  width: 77px;
+  margin-left: -10px;
+  height: 22px;
+  color: #f6f0f0;
+  background-color: #36b37e;
+  border-radius: 8px;
+  text-align: center;
+}
+
+.work-tag-popup {
+  display: flex;
+  font-weight: 600;
+  flex-direction: row;
+  align-items: center;
+  padding: 18px;
+  width: 77px;
+  margin-right: 810px;
+  margin-top: -62px;
+  height: 22px;
+  color: #0e0e0e;
+  border-radius: 8px;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+.mobile-tag {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  width: 84px;
+  height: 22px;
+  margin: 3px;
+  background: #ebecf0;
+  padding: 10px;
+  border-radius: 8px;
+  flex: none;
+  order: 3;
+}
+
+.worktag-popup {
+  margin-top: 50px;
+}
+
+.popup-paragraph {
+  flex: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  width: 876px;
+  height: 151px;
+  left: 24px;
+  top: 888px;
+}
+
+.popup-paragraph p {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 30px;
+  text-align: left;
+}
+
+.card-image-details > img {
+  height: 550px;
+  width: 872px;
+  left: 24px;
+  top: 78px;
+  border-radius: 8px;
+  margin-top: -105px;
+}
+
+.btn-close-detail {
+  background-color: white;
+}
+
+.btn-close-detail > img {
+  display: block;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 9px;
+  margin-right: 11px;
+  margin-top: -101px;
+  border-radius: 8px;
+  position: absolute;
+  width: 16px;
+  height: 15px;
+  right: -1px;
+  background-color: white;
+}
+
+@media (max-width: 768px) {
+  .card-work-detail {
+    box-sizing: border-box;
+    position: fixed;
+    width: 600px;
+    height: 1100px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin-top: 20px;
+    margin-left: 10px;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
+  }
```

Comment:
Regarding the implementation of pop-ups on desktop screens:

- the pop-up needs to be centred on the middle of the page;
- outside of the pop-up we need a layer (maybe another container) that will blur the content of the rest of the page;
- the title needs to be aligned to left and on the right, we need to have the buttons;
- unfortunately, I don't have access to all pop-up features on the desktop either - I only can see the image and the title of the project;

Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/167908579-84470645-ad10-4dcd-8da1-90779c5ad973.png) | ![image](https://user-images.githubusercontent.com/83596135/167908651-761bbd16-826a-4e59-8965-d16c17961449.png)

- [ ] Kindly, please refactor the layout of the pop-up on desktop too; if the pop-up is bigger that the screen, it is not a problem to have a vertical scroll for the pop-up;

Reply:
I will add the Blur feature in a while .. please review the rest :) 👍